api/src/main/java/org/apache/iceberg/transforms/Transform.java

```diff
@@ -124,9 +142,47 @@ default boolean isIdentity() {
    *
    * @param value a transformed value
    * @return a human-readable String representation of the value
+   * @deprecated use {@link #toHumanString(Type, Object)} instead; will be removed in 2.0.0
    */
+  @Deprecated
   default String toHumanString(T value) {
-    return String.valueOf(value);
+    if (value instanceof ByteBuffer) {
+      return TransformUtil.base64encode(((ByteBuffer) value).duplicate());
```

Comment:
why do we need to duplicate the `ByteBuffer` for `base64encode`?

Reply:
That is so this method doesn't modify the original buffer. We could either add that here or in the `base64encode` method.

Reply:
Got it. Looking at the `Base64#encode` method, it change the position of the source buffer. should we just save the position and restore it after the encoding is done?

```
Encodes all remaining bytes from the specified byte buffer into a newly-allocated ByteBuffer using the Base64 encoding scheme. Upon return, the source buffer's position will be updated to its limit; its limit will not have been changed. The returned output buffer's position will be zero and its limit will be the number of resulting encoded bytes.
```

Reply:
nm. this is not on the critical code path. duplicate is simpler.